fdbclient/CommitTransaction.h

```diff
@@ -161,6 +163,7 @@ struct CommitTransactionRef {
 	}
 
 	void clear( Arena& arena, KeyRangeRef const& keys ) {
+		// TODO: check do I need to clear flag here
```

Comment:
You don't need to clear the report_conflicting_keys flag here

Reply:
Thanks. Forgot to delete this comment.